src/client/app/helpers/strings.ts

```diff
@@ -52,7 +53,8 @@ export class Strings {
     static readonly githubLogoutFailed = 'Failed to sign out of GitHub';
     static readonly profileCheckFailed = 'Failed to get the GitHub profile';
     static readonly gistRetrieveFailed = 'Failed to retrieve GitHub gists';
-    static readonly gistShareFailed = 'Failed to share the GitHub gist';
+    static readonly gistShareFailedBody = 'Failed to share the GitHub gist';
+    static readonly gistShareFailedTitle = 'Share failed';
```

Comment:
I'd probably do "Sharing failed" instead of "Share failed".
In general, we've been doing string peer-review by putting `/* NEEDS STRING REVIEW */` next to new strings, and having Michael Saunders then look through them.  Search for this phrase and see other instances where I have it